benchmark/atoi.cpp

```diff
@@ -1,6 +1,8 @@
 #include "zoo/swar/SWAR.h"
 #include "zoo/swar/associative_iteration.h"
 
+#include <immintrin.h>
```

Comment:
might be good to ifdef here based on platform?

Reply:
This is just an experiment.

Reply:
atoi entirely is an experiment?

Reply:
yes, there are several atoi experiments to compare and see what affects performance at a micro level.